x-pack/plugins/security_solution/cypress/e2e/detection_rules/bulk_edit_rules.cy.ts

```diff
@@ -189,7 +189,9 @@ describe('Detection rules, bulk edit', () => {
       clickAddTagsMenuItem();
       waitForMixedRulesBulkEditModal(expectedNumberOfCustomRulesToBeEdited);
 
-      checkPrebuiltRulesCannotBeModified(totalNumberOfPrebuiltRules);
+      getPrepackagedRulesCount().then((prepackagedRulesCount) => {
+        checkPrebuiltRulesCannotBeModified(prepackagedRulesCount);
+      });
```

Comment:
Nit: Let's be consistent with naming and call it `prebuilt` everywhere :)

Reply:
I'd stick to `prepackaged`, then. As that's how we call them on the API level, and that's something we cannot change already.